setup.py

```diff
@@ -1,8 +1,10 @@
+import os
 import re
 
 from setuptools import find_namespace_packages, setup
 
 
+os.system("export PIP_EXTRA_INDEX_URL=https://pip.repos.neuron.amazonaws.com")
```

Comment:
```suggestion
os.environ["PIP_EXTRA_INDEX_URL"] = "https://pip.repos.neuron.amazonaws.com"
```

Reply:
What happens if it was already set by someone?
Can we append?

If not the next best thing we can do is cache the previous value of the variable, set it to what we need, and restore it back to its original value.

Reply:
I just found out that the setuptools have native support for extra index url. Tested with PyPI, works well: https://test.pypi.org/manage/project/optimum-neuron/release/0.0.4.dev1/